doc/source/user/c-info.how-to-extend.rst

```diff
@@ -408,7 +404,6 @@ writeable). The syntax is
 
         Alternatively, the bit-width names can be used as supported on the
         platform. For example:
-
             :c:data:`NPY_INT8`, :c:data:`NPY_INT16`, :c:data:`NPY_INT32`,
```

Comment:
This one and the one on line 401 are needed

Reply:
```suggestion

            :c:data:`NPY_INT8`, :c:data:`NPY_INT16`, :c:data:`NPY_INT32`,
```